src/pages/profile/edit.tsx

```diff
@@ -15,25 +15,14 @@ import withAccessType from '@/lib/hoc/withAccessType';
 import { CookieService, PermissionService } from '@/lib/services';
 import { PrivateProfile } from '@/lib/types/apiResponses';
 import { CookieType, SocialMediaType } from '@/lib/types/enums';
-import { capitalize, fixUrl, getMessagesFromError, getProfilePicture } from '@/lib/utils';
+import { capitalize, fixUrl, getProfilePicture, reportError } from '@/lib/utils';
 import DownloadIcon from '@/public/assets/icons/download-icon.svg';
 import DropdownIcon from '@/public/assets/icons/dropdown-arrow-1.svg';
 import styles from '@/styles/pages/profile/edit.module.scss';
```

Comment:
```suggestion
import styles from '@/styles/pages/EditProfile.module.scss';
```
This isn't caused by your PR but can we flatten all of the page level CSS sheets to one folder by renaming them and fixing the imports because there's not really any reason to nest them in folders like we do with pages to define the routing structure.